test/run-in-travis.sh

```diff
@@ -21,8 +21,26 @@ regression_tests() {
 	sh /usr/share/crmsh/tests/regression.sh
 }
 
-unit_tests
-rc_unittest=$?
-configure
-make_install
-regression_tests && exit $rc_unittest
+bootstrap_tests() {
+        SUFFIX="${1:-*}"
+	echo "** Bootstrap process tests using python-behave"
+        behave --no-logcapture --tags "@bootstrap" --tags "~@wip" /usr/share/crmsh/tests/features/bootstrap_$SUFFIX.feature
```

Comment:
I don't still understand why you are adding this `bootstrap_` prefix here hardcoded. Why are you not using the whole name? I don't see any need to have `bootstrap` prefix in the test files when we are adding this as a tag.

Reply:
I've changed it, change function name as `functional_tests`, and for now it accept 3 types feature files(bootstrap, qdevice, hb_report)